man/systemd-sysext.xml

```diff
@@ -151,7 +151,8 @@
     <command>confext</command> will extend only <filename>/etc</filename>. Files and directories contained
     in the confext images outside of the <filename>/etc/</filename> hierarchy are <emphasis>not</emphasis>
     merged, and hence have no effect when included in the image. Formats for these images are of the
-    same as sysext images.</para>
+    same as sysext images. The merged hierarchy will be mounted with <literal>MS_NOSUID</literal> and
```

Comment:
hmm, given this is user facing, maybe we should call this `nosuid` in the docs, and not `MS_NOSUID`. The latter is a C hacker's name after all...